src/cmd-compress

```diff
@@ -12,10 +12,20 @@ import argparse
 sys.path.insert(0, os.path.dirname(os.path.abspath(__file__)))
 from cmdlib import run_verbose, write_json, sha256sum_file, rm_allow_noent
 
+DEFAULT_COMPRESSION_ALGO = 'gzip'
+
 parser = argparse.ArgumentParser()
 parser.add_argument("--build", help="Build ID")
+parser.add_argument("--algorithm",
```

Comment:
`algorithm` is a misnomer, since the algorithms are respectively LZMA and LZ77.  Something like `--compressor` would have been better.  Probably not worth changing now, though.

Reply:
https://github.com/coreos/coreos-assembler/pull/596